container_pipeline/workers/build.py

```diff
@@ -27,14 +27,16 @@ def handle_job(self, job):
         queue to be processed later. Else, it goes ahead with running
         build for the job.
         """
+        self.job = job
```

Comment:
this wont break the inheritance right? I mean base worker calls the handle build job from run method.

Reply:
No this won't break inheritance. Base worker calls `handle_job` function with a parameter called `job` while we're assigning its value to a instance variable which has value `None`. In fact, I'm planning to move `self.job = None` from workers' `__init__` method to the base class's `__init__` method.